.gitignore

```diff
@@ -13,3 +13,4 @@
 
 # rspec failure tracking
 .rspec_status
+vendor
```

Comment:
This has no relation with rspec.
Please move this to upper line.

And I think your intension will be satisfied by `/vendor/`; it may need to be `vendor/`.